test/test-applications/calltarget/CallTargetNativeTest/integrations.json

```diff
@@ -0,0 +1,1671 @@
+﻿[
+  {
+    "name": "CallTargetTests",
+    "method_replacements": [
+      {
+        "caller": {},
+        "target": {
+          "assembly": "CallTargetNativeTest",
+          "type": "CallTargetNativeTest.With0Arguments",
+          "method": "VoidMethod"
+        },
+        "wrapper": {
+          "assembly": "Datadog.Trace.ClrProfiler.Managed, Version=1.20.0.0, Culture=neutral, PublicKeyToken=def86d061d0d2eeb",
```

Comment:
I have two suggestions on this `integrations.json`:
1. We could generate this `integrations.json` file in `build/tools/PrepareRelease/GenerateIntegrationDefinitions.cs`
2. We could put these classes inside a different assembly so we don't add a lot of clutter to `Datadog.Trace.ClrProfiler.Managed.dll`

Reply:
I created the Noop integrations to help while we create new integrations to check we are intercepting the target method correctly before writing the actual integration. But, if this is no required I could create the integrations on the same CallTargetNativeTest project for the example and remove the NoOp from `Datadog.Trace.ClrProfiler.Managed.dll` . WDYT?

Reply:
Sorry for the delay on this comment. I'd prefer removing the NoOp from `Datadog.Trace.ClrProfiler.Managed.dll` since it's really test-only data and creating the integrations on the CallTargetNativeTest project

Reply:
Ok, will do

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/1007/commits/5aa5b87b46d4e92cab7c3fefc75ccb57a4cbc907